packages/nimble-components/src/table/tests/table.spec.ts

```diff
@@ -478,6 +478,28 @@ describe('Table', () => {
             expect(element.checkValidity()).toBeTrue();
         });
 
+        it('allows row hover styling when not scrolling', async () => {
+            await connect();
+            await element.setData(simpleTableData);
+            await waitForUpdatesAsync();
+            column1.groupIndex = 0;
+            await waitForUpdatesAsync();
+
+            expect(pageObject.isRowHoverStylingEnabled()).toBeTrue();
+        });
+
+        it('does not allow row hover styling while scrolling', async () => {
+            await connect();
+            await element.setData(simpleTableData);
+            await waitForUpdatesAsync();
+            column1.groupIndex = 0;
+            await waitForUpdatesAsync();
+
+            element.viewport.dispatchEvent(new Event('scroll'));
+            await waitForUpdatesAsync();
+            expect(pageObject.isRowHoverStylingEnabled()).toBeFalse();
```

Comment:
If we wanted to check that hover styling got re-enabled when scrolling finished, what would that look like? I'm guessing:
1. dispatch `'scrollend'`
2. wait for a sufficient amount of time for the timers to finish reliably
3. assert

Assuming that step 2 is messy, I'm ok not testing this. But just wanted to make sure we thought about it.

Reply:
I think this is a reasonable test to write. If it ends up being flaky, we can nix it.